src/main/java/com/hedvig/claims/aggregates/ClaimsAggregate.java

```diff
@@ -362,4 +393,25 @@ public void on(EmployeeClaimStatusUpdatedEvent e) {
     this.isCoveringEmployee = e.isCoveringEmployee();
   }
 
+  @EventSourcingHandler
+  public void on(ClaimFileUploadedEvent event) {
+    val claimFile = new ClaimFile();
+      claimFile.setId(event.getClaimFileId());
+      claimFile.setBucket(event.getBucket());
+      claimFile.setKey(event.getKey());
+      claimFile.setContentType(event.getContentType());
+      claimFile.setFileName(event.getFileName());
+      claimFile.setUploadedAt(event.getUploadedAt());
+    claimFiles.add(claimFile);
+  }
+
+  @EventSourcingHandler
+  public void on(ClaimFileMarkedAsDeletedEvent event) {
+    markedAsDeleted = true;
```

Comment:
This should probably be along the lines of:

```suggestion
    claimFiles.find(claim -> claim.id = event.claimId)!!.markedAsDeleted = true 
```